Sources/MapboxMaps/Foundation/Camera/CameraManager.swift

```diff
@@ -464,13 +466,25 @@ public class CameraManager {
             return CLLocationCoordinate2D(latitude: 0, longitude: 0)
         }
 
+        var pitchFactor: CGFloat = mapView.pitch
```

Comment:
It'd be good to get an explanation of what this math is trying to do in the comments here.